docs/core/diagnostics/managed-debuggers.md

```diff
@@ -0,0 +1,33 @@
+---
+title: .NET Core Managed Debuggers
+description: An overview of the Visual Studio and VS Code managed debuggers.
+author: stmaclea
+ms.author: stmaclea
+ms.date: 8/5/2019
+ms.topic: article
+ms.prod: .net-core
+---
+# .NET Core Managed Debuggers
+
+Debuggers allow programs to be paused or executed step-by-step. When paused, the current state of the process can be viewed. By stepping through key sections, you gain understanding of your code and why it produces the result that it does.
+
+Microsoft provides debuggers for managed code in the **Visual Studio** and **Visual Studio Code** integrated development environments.
+
+## Visual Studio Managed Debugger
+
+**Visual Studio** is an IDE with the most comprehensive debugger available. Visual Studio is an excellent default choice for developers working on Windows.
+  - [Tutorial - Debugging a .NET Core application on Windows with Visual Studio](https://docs.microsoft.com/dotnet/core/tutorials/debugging-with-visual-studio)
```

Comment:
Links should be relative to the current article and point to source files unless it points to a different repo.

```suggestion
  - [Tutorial - Debugging a .NET Core application on Windows with Visual Studio](../debugging-with-visual-studio.md)
```